pkg/operator/ceph/file/filesystem.go

```diff
@@ -278,10 +278,16 @@ func downFilesystem(context *clusterd.Context, clusterInfo *cephclient.ClusterIn
 }
 
 // generateDataPoolName generates DataPool name by prefixing the filesystem name to the constant DataPoolSuffix
+// or get predefined name from spec
 func generateDataPoolNames(f *Filesystem, spec cephv1.FilesystemSpec) []string {
```

Comment:
How about a unit test for this name generation method?

Reply:
Sure, let me add it